File: Programmers/Inho/C#/Level01/서울에서 김서방 찾기.cs
public class Solution
{
    public string solution(string[] seoul)
    {
        int index = 0;

        foreach (string kim in seoul)
        {
            if (kim == "Kim")
                break;
            index++;
        }
        string answer = $"김서방은 {index}에 있다";
        return answer;
    }
}

File: Programmers/Inho/C++/Level01/서울에서 김서방 찾기.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

string solution(vector<string> seoul) {
    string kim = "Kim";
    string answer;
    int cnt = 0;
    for(string st: seoul)
    {
        if(kim.compare(st)==0)
            break;
        cnt++;

    }
    answer += "김서방은 ";
    answer += to_string(cnt);
    answer += "에 있다";
    return answer;
}